openedx_webhooks/github/dispatcher/rules/__init__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+r"""
+Rules evaluted by the dispatcher.
+
+Each dispatcher rule determines whether an event requires additional
+processing. Each rule implements the ``run(event_type, event)``
+interface.
+
+Inputs
+------
+
+-  `event\_type`_ is one of the GitHub event types as delivered via the
+   ``X-Github-Event`` header.
+
+-  event is the event payload parsed into a Python ``dict``.
+
+.. _event\_type: https://developer.github.com/v3/activity/events/types/
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from . import github_activity
+
+RULES = (
+    github_activity,
+)
+"""
+Tuple[Module, ...]: A tuple of rules to process, in order
```

Comment:
Is this the new typing syntax? I see these docstring-like strings, which aren't docstrings. Why aren't they comments?

Personally, I'd prefer to use a list for RULES and things like it.  Lists are like arrays, tuples are like structs.